components/brave_wallet_ui/components/buy-send-swap/select-network-with-header/index.tsx

```diff
@@ -27,20 +24,12 @@ export const SelectNetworkWithHeader = ({
   hasAddButton,
   onAddNetwork
 }: Props) => {
-  // custom hooks
-  const { selectSendAsset } = useSend()
-
   // redux
   const dispatch = useDispatch()
 
   // methods
   const onSelectCustomNetwork = React.useCallback((network: BraveWallet.NetworkInfo): void => {
     dispatch(WalletActions.selectNetwork(network))
-
-    selectSendAsset(makeNetworkAsset(network))
-
-    dispatch(PanelActions.navigateTo('main'))
```

Comment:
`dispatch(PanelActions.navigateTo('main'))` was already being called by `onBack()` in the `Panel` container.
also shouldn't have been called from the `Page`